### ISSUE TITLE
Is kebab-case in witness protection?

### ISSUE THREAD

**Issue by sophodoros** [blakeembrey/change-case#151]
Hi. I use your change-case library in my massively popular Code Valet extension for VS Code. (Over 100 users, at least half of which are _not_ immediate family.) This past week while doing a general update of the Valet’s npm libs I found that updating to the latest 4.x version of change-case broke the build because it appeared to no longer support kebab-case. 

However, after a little investigating I found that you hadn’t abandoned kebab-case after all, but instead had renamed it to param-case. Why? Once you get the reference, kebab-case is a lot more fun and descriptive (in English, at least) than the dull and generic param-case. 

Also, why was the name change kept a secret? Maybe I missed something, but except for the fact that you’ve still got a “kebab-case” keyword tag on the change-case npm page, from your repo’s perspective it’s like kebab-case never existed. 

Clearly this is not really an issue for your library. However, the Valet contains a “Kebab case” entry in its tools menu, and it would be nice to know if the Valet could suffer jeers from peers for using a politically incorrect reference. 

Regardless, many thanks for publishing (and maintaining!) the change-case library.

**Comment by blakeembrey** [comment]
I don’t believe I renamed anything but I did remove aliased names, which may be what you’re referring to? I’ve never owned kebab-case on NPM, so that’s why it was called param-case - the other name was already taken. Tags are just a way to find things, since kebab case is a known name for that casing.

**Comment by blakeembrey** [comment]
You can see the package owner here: https://www.npmjs.com/package/kebab-case

**Comment by blakeembrey** [comment]
> However, the Valet contains a “Kebab case” entry in its tools menu, and it would be nice to know if the Valet could suffer jeers from peers for using a politically incorrect reference.

On this point, I wouldn’t worry about it and would personally just alias them in your own code! I think it’s a better name too and I’m not sure it can be construed as politically incorrect - it describe how it looks, just like snake case or camel case.

**Comment by sophodoros** [comment]
Thanks for the explanation and the quick response. I guess if I'd spent a little more time in the wayback machine I would've noticed that param-case pre-dates the 4.0 change-case release and thus wasn't part of some nefarious scheme to replace kebab-case.

So, you can consider this "issue" closed. Once again, many thanks, and happy New Year!
